kbfsdokan/main.go

```diff
@@ -33,14 +34,14 @@ To run against remote KBFS servers:
   kbfsdokan [-debug] [-cpuprofile=path/to/dir]
     [-bserver=%s] [-mdserver=%s]
     [-runtime-dir=path/to/dir] [-label=label] [-mount-type=force]
-    [-log-to-file] [-log-file=path/to/file]
+    [-log-to-file] [-log-file=path/to/file][-service-mount]
     /path/to/mountpoint
```

Comment:
Now "/path/to/mount" is optional, depending on whether "-service-mount" is given, right? So I think it should be surrounded with square brackets, or else have another usage line showing "-service-mount" without an explicit mount passed in.